bin/v-add-backup-host

```diff
@@ -50,6 +50,7 @@ EOF
 
 # Defining sftp command function
 sftpc() {
+    if [ "$privatekey" != "yes" ]; then 
```

Comment:
Code format should be adjusted/reformated. if condition should never be on one line with following code (epxect fi or else).

Reply:
EOF needs to be on the first char otherwise it break everything else has been changed